### DISCUSSION TITLE
Pay in wallet, doesn't work

### DISCUSSION THREAD

**Discussion by MODUNICO** [btcpayserver/btcpayserver#6141] [Technical Support] [answered]
Hi, after clicking "Pay in wallet" a new tab opens, all in white. Why is that?
![Pay in wallet](https://github.com/user-attachments/assets/cfc325c9-17af-416d-9c28-c330b85e95d1)

**Comment by pavlenex** [comment, marked as answer]
If you're opening it on desktop, you need to have desktop bitcoin wallet, if you're opening it on mobile - mobile wallet is required. I would assume you don't have it. To pay you can also just scan a QR code. If you can tell us more on what exactly are you doing we may be able to provide more info on how to troubleshoot.
